configure.ac

```diff
@@ -8022,21 +8027,41 @@ if test "$wxUSE_TESTS_SUBDIR" != "no"; then
     dnl Configure tests directory
     SUBDIRS="$SUBDIRS tests"
 
-    dnl Check for catch (C++ Automated Test Cases in Headers) availability.
-    AC_MSG_CHECKING([whether catch2/catch.hpp exists])
-    if ! test -f "$srcdir/3rdparty/catch/single_include/catch2/catch.hpp" ; then
-        AC_MSG_RESULT([no])
-        AC_MSG_ERROR([
-    CATCH (C++ Automated Test Cases in Headers) is required, the required file
-    $srcdir/3rdparty/catch/single_include/catch2/catch.hpp couldn't be found.
+   dnl Check for catch (C++ Automated Test Cases in Headers) availability.
+    if test "$wxUSE_CATCH2" != "builtin"; then
+        if test -n "$PKG_CONFIG"; then
```

Comment:
We don't test for `PKG_CONFIG` anywhere else, so either we shouldn't do it here neither or we should do it in all the other places too... But is it even possible for it to be empty? I.e. won't configure stop if `PKG_PROG_PKG_CONFIG` doesn't find it?